test/transferSpec.js

```diff
@@ -41,31 +41,15 @@ describe('Plugin transfers (optimistic)', function () {
       assert.isFunction(this.pluginA.send)
     })
 
-    it('should send an optimistic transfer with 0 amount', function (done) {
```

Comment:
0-amount transfers are going to be removed in favor of `sendMessage()` (see https://github.com/interledger/rfcs/issues/96).